ezyrb/ann.py

```diff
@@ -0,0 +1,96 @@
+"""
+Module for Artificial Neural Network (ANN) Prediction.
+"""
+import torch
+import torch.nn as nn
+import numpy as np
+from .approximation import Approximation
+
+class ANN(Approximation):
+    """
+    Feed-Forward Artifical Neural Network (ANN).
+    
+    Example:
+    >>> import ezyrb
+    >>> import torch
+    >>> import torch.nn as nn
+    >>> import numpy as np
+    >>> 
+    >>> x = np.random.uniform(-1, 1, size=(4, 2))
+    >>> y = np.array([np.sin(x[:, 0]), np.cos(x[:, 1]**3)]).T
+    >>> ann = ezyrb.ANN()
+    >>> ann.fit(x,y)
+    >>> y_pred = ann.predict(x)
+    >>> print(y)
+    >>> print(y_pred)
+    """
+        
+    def __init__(self):
+        self.trained_epoch = 0   
+        # number of already trained iterations
```

Comment:
the single comments go above the code line, not below

Reply:
Moreover, I totally suggest to remove such comments, since the class variables should be properly explained in the class documentation. No need for duplication!

Reply:
These comments have been ideated as a guideline. So maybe they could be useful if changes are needed by a new user.

Reply:
That is the exact purpose of the documentation: helping new users. Please remove the comments.